Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/actions/FilterActionOnElementFilter.java

```diff
@@ -55,7 +55,8 @@ public boolean execute() {
                 return false;
             }
         } catch (Exception e) {
-            Messagebox.show("Error in filtering. Error message: " + e.getMessage());
+            // LOGGER.error("Error in filtering. Error message: " + e.getMessage());

```

Comment:
Can this be removed, or better, uncommented? If uncommented, include `e` in the logging call.